.expeditor/builder_seed.txt

```diff
@@ -0,0 +1,14 @@
+# Supervisor and prerequisites
+core/hab-launcher
+core/hab
```

Comment:
should `core/hab-plan-build` and `core/hab-backline` also be included here?  They'll get pulled in as deps, but do we want to be explicit?  

Also, do we want to include the docker exporter for Builder?

Reply:
All those get pulled in as dependencies... I just wanted to keep things simple and high-level.